pom.xml

```diff
@@ -52,12 +52,19 @@
         <avro.version>1.8.2</avro.version>
         <arrow.version>6.0.1</arrow.version>
         <bigquery-connector-common.version>0.23.2</bigquery-connector-common.version>
-        <flink.version>1.12.0</flink.version>
+        <flink.version>1.11.0</flink.version>
+        <flink.connector.test.version>1.12.0</flink.connector.test.version>
         <google-cloud-dataproc.version>2.3.1</google-cloud-dataproc.version>
-        <google-cloud-storage.version>2.3.0</google-cloud-storage.version>
+        <google-cloud-storage.version>2.6.1</google-cloud-storage.version>
         <google-truth.version>1.1.3</google-truth.version>
         <guava.version>30.1.1-jre</guava.version>
         <slf4j.version>1.7.15</slf4j.version>
+		<scala.version>2.12</scala.version>
+        <hadoop.core.version>0.20.2</hadoop.core.version>
```

Comment:
Hadoop version should be 2.10.0

Reply:
Could not find the version 2.10.0 in the mvnrepository. https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-core. So added the latest version as 1.2.1